docs/release/release_0_4_7.md

```diff
@@ -121,6 +121,8 @@ See below for the full list of changes.
 - `layer.status` was deprecated in version 0.4.4 and is removed in this
   version. Instead, use `layer.get_status(viewer.cursor.position, world=True)`
   (#2443).
+- Label-layers must be of integer type, float `Labels` layers are no longer allowed (see #2491)
+- If a magicgui-annotated function took an `napari.layers.Image` as parameter, this included `napari.layers.Labels` layers. Starting with 0.4.7, developers have to change the accepted parameter type to `napari.layers.Layer` if they want to consume images and labels.  
```

Comment:
this one actually belongs in 0.4.6, and results from #2307.  Also "developers have to change the accepted parameter type to `napari.layers.Layer` if they want to consume images and labels" is _slightly_ misleading, because it misses the fact that your dropdown will have _all_ layers (not just images and labels).

Reply:
Something like `typing.Union[Image, Labels]` doesn't work, correct?

Reply:
correct... not yet unfortunately

Reply:
Ok, I moved the note to 0.4.6 and added a comment regarding this aspect